### PR TITLE
DOC-480: Refer to "per second" limits as instantaneous rate limits

### PR DESCRIPTION
**Pull request by m-hulbert**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

This PR attempts to address [this comment](https://github.com/ably/docs/pull/1101#issuecomment-907109874) from the PR introducing limits. The intention is to be consistent with terminology and not refer to instantaneous rate limits as "per second limits".

## Review

This PR can be reviewed by viewing the [limits](http://ably-docs-pr-1199.herokuapp.com/general/limits) page.

### CODE REVIEW

File: content/general/limits.textile [hunk 1]
```diff
@@ -298,57 +298,57 @@ Webhook batch concurrency is the number of webhook batches that can be processed
...
-The Firehose message rate is the maximum amount of Firehose messages that can be streamed per second.
+The Firehose message rate is the maximum rate at which Firehose messages can be streamed.
 
-If the Firehose message rate is exceeded, a "global rate limit":#global-rates on publishing to Firehose is applied.
+If the Firehose message rate is exceeded, a "global instantaneous rate limit":#global-rates on publishing to Firehose is applied.
 
 h3(#function-invocations). Function invocation rate
 
-The function invocation rate is the maximum number of functions that can be invoked per second. This includes AWS Lambda, Google Cloud and Azure functions that can be triggered by "events":https://ably.com/documentation/general/events.
+The function invocation rate is the maximum rate at which functions can be invoked. This includes AWS Lambda, Google Cloud and Azure functions that can be triggered by "events":https://ably.com/documentation/general/events.
```
Comment (marklewin) [inline content/general/limits.textile:307]: Would it be clearer to say "The function invocation rate _limit_ is the maximum rate at which functions can be invoked'? Without the "limit" in there, the function invocation rate is just the number of times you are calling the function. This might also be applicable elsewhere (e.g. Firehose message rate)

**Review by marklewin — CHANGES_REQUESTED** [review]

File: content/general/limits.textile (hunk 1 above)
Comment (m-hulbert) [reply content/general/limits.textile]: Coincidentally I did have that originally but removed it for being too wordy. I've added it back in for all occurrences of rate limits though, as I didn't consider the inference you've pointed out, thanks.

**Review by marklewin — APPROVED** [review]
LGTM!

**Comment by tbedford** [comment]
@paddybyers - do you have feedback before I merge? Thanks!

**Review by paddybyers — APPROVED** [review]
LGTM, thanks
